crates/biome_css_formatter/tests/specs/css/comments/function-argument-comments.css

```
@@ -0,0 +1,5 @@
+.comments{
+value:outerFunctionWithAnExtremelyLongName(1,
```

Stylelint (17.14.0)

[error] Expected "outerFunctionWithAnExtremelyLongName" to be "outerfunctionwithanextremelylongname" (function-name-case)

(function-name-case)